_truststores/00_index.md

```diff
@@ -5,78 +5,74 @@ collection: truststores
 permalink: /truststores/
 ---
 
-This is a high-level overview of what application trust stores are and a list of commonly used application trust stores and settings.  
+This is a high-level overview of what application trust stores are and a list of commonly used trust stores and settings.  
 
 * [What is a Trust Store?](#what-is-a-trust-store)
 * [What are the most commonly used Trust Stores?](#what-are-the-most-commonly-used-trust-stores)
-* [How do I set dynamic path validation for the trust store in Windows operating systems?](#how-do-i-set-dynamic-path-validation-for-the-trust-store-in-windows-operating-systems)
-* [How do I check which Federal PKI certificate policies are trusted by Adobe?](#how-do-i-check-which-federal-pki-certificate-policies-are-trusted-by-adobe)
+* [How do I set dynamic path validation for the Trust Store in Windows operating systems?](#how-do-i-set-dynamic-path-validation-for-the-trust-store-in-windows-operating-systems)
```

Comment:
make trust store lower case everywhere vs upper case

Reply:
Will fix all instances.  Thanks

Reply:
For the main announcement (00_index.md), "trust store" references should **not** be changed to "certificate store"?  Many trust store refs in it.

Reply:
nope; for the main page, the trust store references should not be changed to certificate store.

Reply:
Got it.  Thanks.